examples/shader/custom_phase_item.rs

```diff
@@ -198,18 +198,16 @@ fn main() {
 fn setup(mut commands: Commands) {
     // Spawn a single entity that has custom rendering. It'll be extracted into
     // the render world via [`ExtractComponent`].
-    commands
-        .spawn(SpatialBundle {
-            visibility: Visibility::Visible,
-            transform: Transform::IDENTITY,
-            ..default()
-        })
+    commands.spawn((
+        Visibility::default(),
```

Comment:
Shouldn't this be `Visibility::Visible`?

Reply:
`Visibility::default()` becomes `Visibility::Inherited` which behaves as `Visibility::Visible` if there is no parent hidden afaik, which in this case there isn't.